ecc/Makefile

```diff
@@ -1,11 +1,13 @@
+# Copyright 2019 Intel Corporation
 # Copyright IBM Corp. All Rights Reserved.
 #
 # SPDX-License-Identifier: Apache-2.0
 
 TOP = ..
 include $(TOP)/build.mk
 
-LD_LIBRARY_PATH := LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:$$(pwd)/enclave/lib
+LD_LIBRARY_PATH := LD_LIBRARY_PATH=${LD_LIBRARY_PATH}
```

Comment:
hmm, if we don't define anything, why not just drop it? Seems rather confusing now (and in general might even be problematic, at least i'm not 100% whether the semantics of LD_LIBRARY_PATH is the same of having it undefined vs defined as empty ...

Reply:
it must not be empty cause the sgx sdk libraries are needed.
however, yes, it can be simply removed (also from hte commands) at this point.
done.